examples/config/networks/mantle_testnet.json

```diff
@@ -0,0 +1,21 @@
+{
+  "network_type": "EVM",
+  "slug": "mantle_testnet",
+  "name": "Mantle Sepolia Testnet",
+  "rpc_urls": [
+    {
+      "type_": "rpc",
+      "url": {
+        "type": "plain",
+        "value": "https://rpc.ankr.com/mantle_sepolia"
+      },
+      "weight": 100
+    }
+  ],
+  "chain_id": 5003,
+  "block_time_ms": 14300,
+  "confirmation_blocks": 12,
+  "max_past_blocks": 18,
```

Comment:
If there's no specific reason for these to be hardcoded, let's adjust these `max_past_blocks` fields to the right ones for all the networks. There's a calculation we do:

`(cron_interval_ms/block_time_ms) + confirmation_blocks + 1`

Or simply remove them which defaults to the above calculation.